deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/BaseLayer.java

```diff
@@ -97,8 +93,21 @@ public LayerConfT layerConf() {
         INDArray W = getParamWithNoise(DefaultParamInitializer.WEIGHT_KEY, true, workspaceMgr);
 
         INDArray epsilonNext = workspaceMgr.createUninitialized(ArrayType.ACTIVATION_GRAD, new long[]{W.size(0), delta.size(0)}, 'f');
+        if(hasLayerNorm()) {
+            INDArray g = getParamWithNoise(DefaultParamInitializer.GAIN_KEY, true, workspaceMgr);
```

Comment:
Does it make sense to apply weight noise (dropconnect, gaussian noise etc) on the gain parameter?
Probably not IMO, especially given it's already applied to weights etc.

Reply:
You are right, it shouldn't be applied.